### PR TITLE
chore: update and apply linters

### PR DESCRIPTION
**Pull request by pat-s**

### CODE REVIEW

File: .woodpecker/lint.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+when:
+  - event: tag
+  - event: push
+    branch: main
+  - event: pull_request
+
+steps:
+  lint-markdown:
+    depends_on: []
+    image: davidanson/markdownlint-cli2:v0.14.0
```
Comment (xoxys) [inline .woodpecker/lint.yaml:10]: I would still prefer full qualified image names everywhere.

File: .ecrc [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (xoxys) [inline .ecrc:1]: .ecrc file is deprecated see https://github.com/editorconfig-checker/editorconfig-checker?tab=readme-ov-file#configuration

File: Makefile [hunk 3]
```diff
@@ -8,6 +8,22 @@
...
+
+lint-editorconfig:
+	editorconfig-checker .
+
+lint-helm:
+	helm lint --with-subcharts charts/*
+
+.PHONY: lint
+lint:
+	make lint-helm && make lint-yaml && make lint-markdown && make lint-editorconfig
```
Comment (xoxys) [inline Makefile:25]:
```suggestion
.PHONY: lint
lint: lint-helm lint-yaml lint-markdown lint-editorconfig
```

File: .ecrc (hunk 2 above)
Comment (pat-s) [reply .ecrc]: Seems like this is not yet supported in the latest release - still made the change and passing the config file explicitly.

**Review by xoxys — APPROVED** [review]

**Pull request merged**